### DISCUSSION TITLE
Is there a good way to make the window size fit automatically?

### DISCUSSION THREAD

**Discussion by Kalmaegi** [tauri-apps/tauri#6274] [Q&A]
I have several styles inside my app and each time I switch to display a different div, is there any easy way to automatically make the size of the app match the size of the displayed div after switching? thanks！

**Comment by FabianLars** [comment]
So there is no automatic way in the sense of "let tauri handle everything itself", but you can [get the div size](https://stackoverflow.com/a/15615701/13375311) each time you change the view, and the use https://tauri.app/v1/api/js/window#setsize to set the window size.
